plugins/generator-1.21.1/neoforge-1.21.1/procedures/utils/projectiles/throwable.java.ftl

```diff
@@ -0,0 +1,9 @@
+private static Projectile initThrowableProjectile(Projectile entityToSpawn, Entity shooter, Vec3 acceleration) {
+	if (shooter != null)
+		entityToSpawn.setOwner(shooter);
+	if (Vec3.ZERO.equals(acceleration)) {

```

Comment:
I am not sure if this makes sense. So only if acceleration is zero, it will be set. Was this PR tested at all?